scripts/zones/Buburimu_Peninsula/mobs/Wake_Warder_Wanda.lua

```diff
@@ -0,0 +1,48 @@
+-----------------------------------
+-- Area: Buburimi Peninsula
+--  MOB: Wake Warder Wanda
+-----------------------------------
+
+require("scripts/globals/status");
+
+-----------------------------------
+-- onMobInitialize Action
+-----------------------------------
+
+function onMobInitialize(mob)
+    mob:setMobMod(MOBMOD_MAGIC_COOL, 50); -- just one spell to spam
+end;
+
+-----------------------------------
+-- onMobSpawn Action
+-----------------------------------
+
+function onMobSpawn(mob)
+    mob:addMod(MOD_SILENCERES,20);
+    mob:addMod(MOD_BINDRES,20);
```

Comment:
mob would be super resistant after going down a few times, should put this in onMobInitialise if that's not supposed to happen (maybe mob_spawn_mods)